README.md

```diff
@@ -1,13 +1,19 @@
-# Azure App Services for Visual Studio Code
+# Azure App Services for Visual Studio Code (Preview)
 The Azure App Services extension for VS Code lets you quickly browse, create, manage, and deploy Azure App Service websites.
 
 ## Features
 
-* Browse sites across all of your Azure subscriptions
-* Create new sites
-* Deploy to your site
-* Start, stop, and restart the site
-* Browse to the Azure Portal for advanced tasks, such as scaling
+### Browse sites across all of your Azure subscriptions
```

Comment:
I feel like the section header should only be used in conjunction with a picture or gif. If everything uses `###`, then I feel like it's screaming at me. Can we change the features without pictures back to a bulleted list?